.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
whatever file or folder names you put in the `.gitignore` file won't be checked into git. It's very common practice to do this for your dependency folders.